src/screens/Departures/StopPlaceView.tsx

```diff
@@ -61,14 +62,26 @@ export default function StopPlaceView({
           }
           sections={quayListData}
           keyExtractor={(item) => item.id}
-          renderItem={({item}) => (
-            <QuaySection
-              quay={item}
-              data={state.data}
-              navigateToDetails={navigateToDetails}
-              navigateToQuay={navigateToQuay}
-            />
-          )}
+          renderItem={({item, index}) => {
+            const renderedItem = (
+              <QuaySection
+                quay={item}
+                data={state.data}
+                navigateToDetails={navigateToDetails}
+                navigateToQuay={navigateToQuay}
+              />
+            );
+            if (index === 0) {
+              return (
+                <>
+                  {renderedItem}
+                  <Feedback mode="departures" quayListData={quayListData} />
+                </>
+              );
+            } else {
+              return renderedItem;
+            }
+          }}
```

Comment:
Med på logikken her, men kanskje dette er mer leselig?

```suggestion
          renderItem={({item, index}) => (
            <>
              <QuaySection
                quay={item}
                data={state.data}
                navigateToDetails={navigateToDetails}
                navigateToQuay={navigateToQuay}
              />
              {index === 0 && (
                <Feedback mode="departures" quayListData={quayListData} />
              )}
            </>
```

Reply:
Tror meningen var å unngå Fragments der det ikke er nødvendig, men det ble litt renere, så tar den med 👍